docs/azure_jumpstart_hcibox/AKS/_index.md

```diff
@@ -10,41 +10,64 @@ Azure Stack HCI can provide host infrastructure for [Azure Kubernetes Service cl
 
-- We can also view Kubernetes resources on the cluster from Azure portal. To do this you will need a service account bearer token. You can create one by opening the C:\HCIBox folder and running the GetServiceAccountBearerToken.ps1 script by right clicking on it and selecting "Run with PowerShell".
+  ![Screenshot showing aks extension](./custom_location_resources.png)
 
-  ![Screenshot showing how to run the service account bearer token generating script](./run_token_script.png)
+Access to the AKS cluster is [managed through Azure RBAC](https://learn.microsoft.com/azure/aks/hybrid/aks-create-clusters-cli#before-you-begin). Create a new Microsoft Entra group or use an existing one where you are a member for this exercise.
 
-- After the script completes it will pause to allow you to copy the token. Highlight the token and right click to copy it.
+- [Create a Microsoft Entra group](https://learn.microsoft.com/entra/fundamentals/how-to-manage-groups) or use an existing one.
 
-  ![Screenshot showing the script running](./run_token_script_result.png)
+- Retrieve the object id of the Microsoft Entra group and copy it down for use in the next step.
 
-- Navigate to the workload cluster in Azure portal and select Namespaces under Kubernetes resources (preview). After signing in with the token your Kubernetes resources will be available to view from Azure portal.
+  ![Screenshot showing Entra group id](./entra_group_id.png)
 
-  ![Screenshot showing Kubernetes token page in Azure portal](./enter_token_portal.png)
+- From the _HCIBox-Client_ virtual machine, open File Explorer and navigate to the "C:\HCIBox" folder. Right-click on "Configure-AksWorkloadCluster.ps1" and then select Open with Code
 
-  ![Screenshot showing Kubernetes resources in Azure portal](./k8s_resources_portal.png)
+  ![Screenshot showing edit with VSCode](./open_with_code.png)
 
-- We can deploy configurations for this workload cluster automatically from source using [GitOps](https://learn.microsoft.com/azure/azure-arc/kubernetes/tutorial-use-gitops-connected-cluster). HCIBox includes a script that will automatically configure a simple "Hello Arc" application and ingress controller on the AKS cluster.
+- Uncomment line 6 in the script and edit the placeholder value for the $aadgroupID parameter with the object ID of your Microsoft Entra group. Save the script and close VSCode when finished.
 
-- Open up Windows Explorer on the _HCIBox-Client_ VM and navigate to the _C:\HCIBox_ folder. From here, right click on the _Deploy-GitOps.ps1_ file and run with PowerShell.
+  ![Screenshot showing editing script](./edit_script.png)
 
-  ![Screenshot showing starting GitOps script](./deploy_gitops.png)
+- - From the _HCIBox-Client_ virtual machine, open File Explorer and navigate to the "C:\HCIBox" folder. Right-click on "Configure-AksWorkloadCluster.ps1" and then select "Run with PowerShell."
```

Comment:
This could in theory be automated, but we want the user to perform the task to learn/see what is happening?

Reply:
yes, and they need to create or be added to a Microsoft Entra group. I felt this was not always automatable.

Reply:
based on their permissions.